packages/web-app-files/src/views/Favorites.vue

```diff
@@ -25,7 +25,11 @@
           <quick-actions class="oc-visible@s" :item="props.resource" :actions="app.quickActions" />
         </template>
         <template #contextMenu="{ resource }">
-          <context-actions v-if="isHighlightedFile(resource)" :item="resource" />
+          <context-actions
+            v-if="!areMultipleSelected && isHighlightedFile(resource)"
+            :item="resource"
+          />
+          <batch-actions v-if="areMultipleSelected" :in-context-actions="true" />
```

Comment:
Please also check if the clicked resource is part of the selected files. Otherwise you'd create the batch actions in the DOM tree for each and every row (same for the other views).

Reply:
Uh yeah, great one!

Reply:
Pretty sure this has been resolved by you (or was it me? 🤔 ) by now, marking as resolved